13-collabs.Rmd

```diff
@@ -5,6 +5,248 @@
+```
+### Step 1. An problem identified/issue created
+
+[GitHub issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) are where we can note, plan, and discuss work for a project. It might first be filed by one person but then used to further describe a problem or feature request. Once the issue is scoped and defined enough it can be assigned to someone who can work on it
+
+### Step 2. Proposing a solution
+
+Creating a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) is proposing a solution that addresses the original issue. The author of the pull request creates a new branch with the necessary changes to address the issue. The pull request description should reference the original issue so we know what it is addressing and explain how the work addresses it.
+
+### Step 3. Reviewer looks at proposed solution
+
+Now the reviewer is given a chance to look at the proposed solution and approve and otherwise improve it in collaboration with the original author.
+
+### Step 4. Refined solution is deployed
+
+Once both the author and the reviewer agree the work is ready its merged and deployed to main! Yay! Time to celebrate.
+
+Now the process can be repeated for a next issue!
+
+## Issues should tell us:
+- What the problem is - including examples or screenshots to demonstrate the problem.
+- What solution(s) could address the problem?
+- Who might be assigned to addressing this problem?
+- Potentially what timeline or urgency the problem has.
+
+## Pull Requests should tell us:
+- What issue is the work addressing?
+- How is it addressing the issue?
+- What pitfalls exist?
+- What should be looked at and reviewed carefully.
+
+## Engaging in Code Review - as an author
+
+The only way to know if your analysis is truly reproducible is to send it to someone else to reproduce! That sentiment is at the heart of code review.
+
+@Parker2017 describes code review:
+
+> Code review will not guarantee an accurate analysis, but it’s one of the most reliable ways of establishing one that is more accurate than before.
+
+Not only does code review help boost the accuracy and reproducibility of the analysis, it also helps everyone involved in the process learn something new!
+
+An effective code review atmosphere is something that individuals and their team have to _commit_ to (pun intended). Effective code review brings so many benefits not only to your project quality but also your communication skills through fostering a learning atmosphere!
+
+In this chapter we will discuss the two sides of code review. Code review ideally includes at least two people: the author of the pull request and the reviewer of the pull request. Depending on your job context, we realize that sometimes authors have to become their own reviewers if code review is not something that can be prioritized by your institution or team.
```

Comment:
```suggestion
In this chapter we will discuss the two sides of code review. Code review ideally includes at least two people: the author of the pull request and the reviewer of the pull request. Depending on your job context, we realize that sometimes authors have to become their own reviewers. This might happen if your team is small or the authors are the only people with a particular skill set.
```

Reply:
explaining why prioritization might not happen